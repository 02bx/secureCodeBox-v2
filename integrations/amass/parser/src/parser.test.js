const fs = require('fs');
const util = require('util');

// eslint-disable-next-line security/detect-non-literal-fs-filename
const readFile = util.promisify(fs.readFile);

const { parse } = require('./parser');

test('example parser parses empty json to zero findings', async () => {
  const fileContent = await readFile(
    __dirname + '/__testFiles__/securecodebox.io.jsonl',
    {
      encoding: 'utf8',
    }
  );

  const res = await parse(fileContent);

  expect(res).toMatchSnapshot();
});
