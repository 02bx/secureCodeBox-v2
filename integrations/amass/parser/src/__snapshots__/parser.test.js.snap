// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example parser parses empty json to zero findings 1`] = `
Array [
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "grafana.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain grafana.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "grafana.securecodebox.io",
    "name": "grafana.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "kibana.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain kibana.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "kibana.securecodebox.io",
    "name": "kibana.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "build.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain build.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "build.securecodebox.io",
    "name": "build.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "monitoring.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain monitoring.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "monitoring.securecodebox.io",
    "name": "monitoring.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "ui.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain ui.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "ui.securecodebox.io",
    "name": "ui.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "juiceshop.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain juiceshop.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "juiceshop.securecodebox.io",
    "name": "juiceshop.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "wpscan.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain wpscan.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "wpscan.securecodebox.io",
    "name": "wpscan.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "hack.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain hack.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "hack.securecodebox.io",
    "name": "hack.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "spider.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain spider.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "spider.securecodebox.io",
    "name": "spider.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "www.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain www.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "www.securecodebox.io",
    "name": "www.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "rancher.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain rancher.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "rancher.securecodebox.io",
    "name": "rancher.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "backend.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain backend.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "backend.securecodebox.io",
    "name": "backend.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "zap.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain zap.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "zap.securecodebox.io",
    "name": "zap.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "bodgeit.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain bodgeit.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "bodgeit.securecodebox.io",
    "name": "bodgeit.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "api-management.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain api-management.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "api-management.securecodebox.io",
    "name": "api-management.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "dvwa.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain dvwa.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "dvwa.securecodebox.io",
    "name": "dvwa.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "prometheus.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain prometheus.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "prometheus.securecodebox.io",
    "name": "prometheus.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "engine.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain engine.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "engine.securecodebox.io",
    "name": "engine.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "logging.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain logging.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "logging.securecodebox.io",
    "name": "logging.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "petstoreapi.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain petstoreapi.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "petstoreapi.securecodebox.io",
    "name": "petstoreapi.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "jenkins.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain jenkins.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "jenkins.securecodebox.io",
    "name": "jenkins.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "bridge.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain bridge.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "bridge.securecodebox.io",
    "name": "bridge.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "petstore.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain petstore.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "petstore.securecodebox.io",
    "name": "petstore.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "sslyze.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain sslyze.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "sslyze.securecodebox.io",
    "name": "sslyze.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "scanner.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain scanner.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "scanner.securecodebox.io",
    "name": "scanner.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "sieve.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain sieve.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "sieve.securecodebox.io",
    "name": "sieve.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "securecodebox.io",
      "source": "Forward DNS",
      "tag": "dns",
    },
    "category": "Subdomain",
    "description": "Found subdomain securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "securecodebox.io",
    "name": "securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "gateway.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain gateway.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "gateway.securecodebox.io",
    "name": "gateway.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "sso.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain sso.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "sso.securecodebox.io",
    "name": "sso.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "test.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain test.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "test.securecodebox.io",
    "name": "test.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "elasticsearch.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain elasticsearch.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "elasticsearch.securecodebox.io",
    "name": "elasticsearch.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "arachni.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain arachni.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "arachni.securecodebox.io",
    "name": "arachni.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "ctf.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain ctf.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "ctf.securecodebox.io",
    "name": "ctf.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "nmap.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain nmap.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "nmap.securecodebox.io",
    "name": "nmap.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "discovery.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain discovery.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "discovery.securecodebox.io",
    "name": "discovery.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "demo.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain demo.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "demo.securecodebox.io",
    "name": "demo.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "dashboard.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain dashboard.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "dashboard.securecodebox.io",
    "name": "dashboard.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "consul.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain consul.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "consul.securecodebox.io",
    "name": "consul.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "target.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain target.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "target.securecodebox.io",
    "name": "target.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "vault.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain vault.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "vault.securecodebox.io",
    "name": "vault.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "docs.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain docs.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "docs.securecodebox.io",
    "name": "docs.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "nikto.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain nikto.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "nikto.securecodebox.io",
    "name": "nikto.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "api-backend.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain api-backend.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "api-backend.securecodebox.io",
    "name": "api-backend.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
  Object {
    "attributes": Object {
      "addresses": Array [
        Object {
          "asn": 24940,
          "cidr": "138.201.0.0/16",
          "desc": "HETZNER-AS ",
          "ip": "138.201.126.99",
        },
      ],
      "domain": "securecodebox.io",
      "name": "api.securecodebox.io",
      "source": "Entrust",
      "tag": "cert",
    },
    "category": "Subdomain",
    "description": "Found subdomain api.securecodebox.io",
    "id": "49bf7fd3-8512-4d73-a28f-608e493cd726",
    "location": "api.securecodebox.io",
    "name": "api.securecodebox.io",
    "osi_layer": "NETWORK",
    "severity": "INFORMATIONAL",
  },
]
`;
